tests/lib/prepare-project.sh

```diff
@@ -66,6 +66,42 @@ fedora_build_rpm() {
     rm $GOPATH/*.src.rpm
 }
 
+opensuse_build_rpm() {
+    release=$(echo "$SPREAD_SYSTEM" | awk '{split($0,a,"-");print a[2]}')
```

Comment:
Could this just come from `/etc/os-release`? I'm not sure why we're encoding it in SPREAD_SYSTEM